docs/LightningNetwork.md

```diff
@@ -44,122 +45,146 @@ export BTCPAYGEN_LIGHTNING="implementationgoeshere"
+- For **LND** use `export BTCPAYGEN_LIGHTNING="lnd"`
+- For **eclair** use `export BTCPAYGEN_LIGHTNING="eclair"` AND `export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-txindex"`
 
 Finally, to begin using Lightning, your blockchain needs to be fully synced.
 
-## Connecting your internal Lightning Node in BTCPay
+# Lightning node configuration in BTCPay
+## Connecting your internal Lightning Node
 
-Regardless of the implementation (c-lightning or LND) you've decided to use, the process of **connecting your internal Lightning Node in BTCPay Server** is the same.
+Irrespective of the LN implementation deployed, the process of connecting your internal Lightning Node in BTCPay Server is the same.
 
-1. If you do not have a store, create one.
-2. Store > Settings > Lightning > Modify
-3. Select "Use internal node"
-4. Click "Save"
-5. See "BTC Lightning node updated" message
-6. Make sure it is enabled
+1. Choose a store
+2. Go to "Lightning" > Select "Use internal node"
+4. Click "Save" > See "BTC Lightning node updated" message
+6. Go to "Public Node Info" > The node should appear **"Online"**
 
-**Your blockchain needs to be fully synced before you try to connect your internal Lightning Node, otherwise the connection will fail.**
 
-![BTCPay Server Lightning Network Setup 1](./img/lightning-node-setup/LightningNetworkNodeSetup1.jpg "BTCPay Server Lightning Network Setup 1")
+![LightningNetworkNodeSetupOverview](./img/lightning-node-setup/LightningNetworkNodeSetupOverview.jpg "BTCPay Server Lightning Network Setup Overview")
 
-![BTCPay Server Lightning Network Setup 2](./img/lightning-node-setup/LightningNetworkNodeSetup2.png "Use the internal node")
+If the internal connection fails, confirm:
+1. The Bitcoin on-chain node is fully synchronized
+2. The Internal lightning node is "Enabled" under "Lightning" > "Settings" > "BTC Lightning Settings"
 
-To ensure that your Lightning node is connected and accessible for use in your store's invoices, try to test your Lightning connection by clicking the "Public Node Info" link. Verify that a successful connection is displayed. If your internal node connection fails, make sure you have a Lightning implementation [enabled](./FAQ/LightningNetwork.md#i-previously-installed-btcpayserver-without-lightning-can-i-enable-it) on your server.
-
-If you are unable to connect to your Lightning node, try [restarting your server](./FAQ/ServerSettings.md#how-to-restart-btcpay-server) or reviewing our [troubleshooting guide](./Troubleshooting.md). You will not be able to accept lightning payments in your store until your test connection shows as successful.
+If you are unable to connect to your Lightning node, try [restarting your server](./FAQ/ServerSettings.md#how-to-restart-btcpay-server) or reviewing our [troubleshooting guide](./Troubleshooting.md). You will not be able to accept lightning payments in your store until your Lightning node appears "Online".
```

Comment:
```suggestion
If you are unable to connect to your Lightning node, try [restarting your server](./FAQ/ServerSettings.md#how-to-restart-btcpay-server) or reviewing our [troubleshooting guide](./Troubleshooting.md). You will not be able to accept lightning payments in your store until your Lightning node appears "Online". Try to test your Lightning connection by clicking the "Public Node Info" link.
```
Having the user to test his connection isn't a bad step, consider adding it back in. ^^